jboss-image-streams.json

```diff
@@ -444,9 +444,10 @@
                     {
                         "name": "1.0",
                         "annotations": {
-                            "description": "OpenJDK S2I images.",
+                            "openshift.io/display-name": "OpenJDK 8",
```

Comment:
Do we want "Red Hat OpenJDK 8?"  Just wondering if we want to distinguish it from other OpenJDK 8 images.